src/Camp24Devon/Inventory.elm

```diff
@@ -0,0 +1,106 @@
+module Camp24Devon.Inventory exposing (..)
+
+import AssocList
+import Camp24Devon.Product as Product
+import PurchaseForm exposing (..)
+import Types exposing (..)
+
+
+maxAttendees =
+    80
+
+
+maxForAccommodationType t =
```

Comment:
I have a workable plan with 16 doubles, 6 singles and 3 dorms - so leaving some wiggle room, maybe we can shift this inventory to:

6 single
30 double
12 group

Reply:
That would involve shifing one of the doubles I identified into a 3 dorm or having a couple of the dorms be 4 rather than 3.

Reply:
Given this year we’re selling rooms not capacity, can you give me the numbers in rooms? I’m guessing 30 doubles meant 30 people so 15 rooms?

Reply:
Yes. 6 singles, 15 doubles and 4 dorms of 3... or 16 doubles and 3 dorms of 4 :)

Reply:
Okay lets go for more dorms and we can always release another double instead if they don't take.

I've just put the campsite spots to 20 – we probably need to come up with a definition of what a "spot" is – I'm going to hazard a guess  we have enough room for 20 of whatever maximal definition that is for now.

Reply:
Sounds good. The designated camping bit is a big lawn. We'll be able to work it out. I'd be suprised if 20 tents arrived too. I'm expecting less than 10.